apps/frontend/modules/scrutin/actions/actions.class.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * scrutins components.
+ *
+ * @package    cpc
+ * @subpackage scrutin
+ */
+class scrutinActions extends sfActions
+{
+  public function executeParlementaire(sfWebRequest $request) {
+    $this->parlementaire = Doctrine::getTable('Parlementaire')->findOneBySlug($request->getParameter('slug'));
+    $this->forward404Unless($this->parlementaire);
+    $this->response->setTitle("Votes de ".$this->parlementaire->nom);
```

Comment:
Utilise plutôt `myTools::setPageTitle($titre, $this->response);` ça formatera avec "NosDéputés.fr - " devant notamment

Reply:
Fait :+1: